### ISSUE TITLE
5.12.15 Upgrade Fails - DATABASE_ERROR: Updating indices mapping fails

### ISSUE THREAD

**Issue by MaxwellDPS** [LuatixHQ/opencti#5399]
## Description

Upgrade from 5.12.4 -> 5.12.15 causes mapping error on initial launch

```
ResponseError: mapper_parsing_exception
	Caused by:
		mapper_parsing_exception: normalizer [string_normalizer] not found for field [keyword]
	Root causes:
		mapper_parsing_exception: normalizer [string_normalizer] not found for field [keyword]
    at DLt.request (/opt/opencti/build/node_modules/@elastic/transport/src/Transport.ts:553:17)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at UBt.putMapping (/opt/opencti/build/node_modules/@elastic/elasticsearch/src/api/api/indices.ts:1338:12)
    at elUpdateIndicesMappings (/opt/opencti/build/src/database/engine.js:806:5)
    at mNu.up (/opt/opencti/build/src/migrations/1701354091187-file_mappings.js:5:3)
```

## Environment

1. OS (where OpenCTI server runs): CentOS Stream 9
2. OpenCTI version: 5.12.15
3. OpenCTI client: Platform
4. Other environment details: Clustered k8s deployment

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Upgrade from 5.12.4 to 5.12.15
2. Check for crash

## Expected Output

A working migrated database

## Actual Output

```json
{
  "category": "APP",
  "errors":
    [
      {
        "attributes": { "genre": "TECHNICAL", "http_status": 500 },
        "message": "Updating indices mapping fail",
        "name": "DATABASE_ERROR",
        "stack": "DATABASE_ERROR: Updating indices mapping fail\n    at error (/opt/opencti/build/src/config/errors.js:8:10)\n    at DatabaseError (/opt/opencti/build/src/config/errors.js:58:48)\n    at /opt/opencti/build/src/database/engine.js:807:13\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at elUpdateIndicesMappings (/opt/opencti/build/src/database/engine.js:806:5)\n    at mNu.up (/opt/opencti/build/src/migrations/1701354091187-file_mappings.js:5:3)",
      },
      {
        "message": "mapper_parsing_exception\n\tCaused by:\n\t\tmapper_parsing_exception: normalizer [string_normalizer] not found for field [keyword]\n\tRoot causes:\n\t\tmapper_parsing_exception: normalizer [string_normalizer] not found for field [keyword]",
        "name": "ResponseError",
        "stack": "ResponseError: mapper_parsing_exception\n\tCaused by:\n\t\tmapper_parsing_exception: normalizer [string_normalizer] not found for field [keyword]\n\tRoot causes:\n\t\tmapper_parsing_exception: normalizer [string_normalizer] not found for field [keyword]\n    at DLt.request (/opt/opencti/build/node_modules/@elastic/transport/src/Transport.ts:553:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at UBt.putMapping (/opt/opencti/build/node_modules/@elastic/elasticsearch/src/api/api/indices.ts:1338:12)\n    at elUpdateIndicesMappings (/opt/opencti/build/src/database/engine.js:806:5)\n    at mNu.up (/opt/opencti/build/src/migrations/1701354091187-file_mappings.js:5:3)",
      },
    ],
  "level": "error",
  "message": "Updating indices mapping fail",
  "timestamp": "2024-01-04T22:59:54.496Z",
  "version": "5.12.15",
}
```

## Additional information

99% sure we have an issue from when we worked with yall last

**Comment by SamuelHassine** [comment]
Hello Max,

We already saw this error for one of our customers. It was linked to the fact that they have indices where the index template was not properly applied (for instance, if you have an index template opencti_stix_domain_objects applying on opencti_stix_domain_objects* but the index is opencti_stix_domain_objects (so * is not matching any suffix).

Can you please check the mapping settings of your indices and confirm that the string_normalizer is everywhere?

Kind regards,
Samuel

**Comment by MaxwellDPS** [comment]
@SamuelHassine As far as I can tell they are all in place, I had @richard-julien on a call when I initially set this up under y'alls recommendations.

**Apologies for anyone else encountering this we are abandoning our platform and wont be debugging any further.**

**_The lack of any in-depth assessment or acknowledgement of the issues here is disheartening at best._**  Y'all havent done anything to help as we have been asking for help and raising the red flag here for a long time #4936 case and point. 

In previous issues here I have tried to provide everything I can on my end, and its always "It'll be fixed in the next update" or "This is specific to you". 

Y'all dont seem to prioitize community support as much these days (or not at all if "This is specific to you") and its getting to be concerning. There has been a distinct shift away from looking into issues for anyone not paying even if it may be more broadly detrimental. I really do appreciate this project and team and have been around this project for a while, but this cultural shift has been noticeable from the non-enterprise side and is frustrating.

TLDR; Doing a complete rebuild as there is 110% an data structure issue and after 6 months of begging for help
